notifier/templates/digest-email-flagged.html

```diff
@@ -0,0 +1,124 @@
+{% load i18n %}
```

Comment:
This was just taken straight from your initial PR, which looks to be pretty much copy pasted from the regular digest email template